velox/connectors/hive/CMakeLists.txt

```diff
@@ -26,11 +26,14 @@ add_library(
 target_link_libraries(
   velox_hive_connector
   velox_connector
+  velox_dwio_catalog_fbhive
   velox_dwio_dwrf_reader
   velox_dwio_dwrf_writer
+  velox_dwio_parquet_reader
+  velox_dwio_parquet_writer
   velox_file
   velox_hive_partition_function
-  velox_dwio_catalog_fbhive)
+  velox_s3fs)
```

Comment:
Missing library "curl" required by AWS.  I applied your patch and the 2 fixes from my PR also needed and I get the following build error:

/usr/bin/ld: /usr/local/lib/libaws-cpp-sdk-core.so: undefined reference to `curl_global_cleanup@CURL_OPENSSL_4'
/usr/bin/ld: /usr/local/lib/libaws-cpp-sdk-core.so: undefined reference to `curl_easy_getinfo@CURL_OPENSSL_4'
/usr/bin/ld: /usr/local/lib/libaws-cpp-sdk-core.so: undefined reference to `curl_global_init@CURL_OPENSSL_4'
/usr/bin/ld: /usr/local/lib/libaws-cpp-sdk-core.so: undefined reference to `curl_easy_perform@CURL_OPENSSL_4'
/usr/bin/ld: /usr/local/lib/libaws-cpp-sdk-core.so: undefined reference to `curl_easy_init@CURL_OPENSSL_4'
/usr/bin/ld: /usr/local/lib/libaws-cpp-sdk-core.so: undefined reference to `curl_easy_pause@CURL_OPENSSL_4'
/usr/bin/ld: /usr/local/lib/libaws-cpp-sdk-core.so: undefined reference to `curl_version_info@CURL_OPENSSL_4'
/usr/bin/ld: /usr/local/lib/libaws-cpp-sdk-core.so: undefined reference to `curl_slist_free_all@CURL_OPENSSL_4'
/usr/bin/ld: /usr/local/lib/libaws-cpp-sdk-core.so: undefined reference to `curl_easy_strerror@CURL_OPENSSL_4'
/usr/bin/ld: /usr/local/lib/libaws-cpp-sdk-core.so: undefined reference to `curl_easy_setopt@CURL_OPENSSL_4'
/usr/bin/ld: /usr/local/lib/libaws-cpp-sdk-core.so: undefined reference to `curl_slist_append@CURL_OPENSSL_4'
/usr/bin/ld: /usr/local/lib/libaws-cpp-sdk-core.so: undefined reference to `curl_easy_cleanup@CURL_OPENSSL_4'
/usr/bin/ld: /usr/local/lib/libaws-cpp-sdk-core.so: undefined reference to `curl_easy_reset@CURL_OPENSSL_4'
collect2: error: ld returned 1 exit status
[687/687] Linking CXX executable velox/functions/prestosql/benchmarks/velox_functions_prestosql_benchmarks_cardinality
ninja: build stopped: subcommand failed.
make[1]: *** [Makefile:87: build] Error 1
make[1]: Leaving directory '/home/pdamons/code/velox-test'
make: *** [Makefile:94: release] Error 2

Reply:
Likely related to your fixes. CI is passing without any build failures.